build.ps1

```diff
@@ -291,6 +306,11 @@ function Build-WiXProject()
   Check-LastExitCode
 }
 
+function Write-PListFile($Path, $Key, $Value)
```

Comment:
This isn't generic, please undo this.  Alternatively, it should take a JSON blob and serialise the into a plist.

Reply:
Bah, it was good enough for our needs. I'd need to serialize into python so I'll revert

Reply:
In fact I just wanted to break the long line, so I did that with a backtick